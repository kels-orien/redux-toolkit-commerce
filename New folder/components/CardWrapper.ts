import styled from 'styled-components'

const CardWrapper = styled.div`
align-items: center;
display: flex;
flex-wrap: wrap;
justify-content: center;
`

export default CardWrapper