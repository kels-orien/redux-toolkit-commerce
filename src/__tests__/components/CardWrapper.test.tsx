import React from 'react'
import renderer from 'react-test-renderer'
import CardWrapper from '../../components/CardWrapper'

test('CardWrapper render correctly', () => {
    const tree = renderer.create(<CardWrapper/>).toJSON();

    expect(tree).toMatchSnapshot()

})