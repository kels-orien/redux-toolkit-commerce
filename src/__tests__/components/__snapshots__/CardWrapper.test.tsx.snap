// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardWrapper render correctly 1`] = `
<div
  className="sc-bdnxRM cejDTm"
/>
`;
